src/components/WalletConnect/NanoContract/NanoContractMethodArgs.js

```diff
@@ -79,11 +77,6 @@ export const NanoContractMethodArgs = ({ blueprintId, method, ncArgs }) => {
       return ncArgs.map((arg, idx) => [methodInfo.args[idx].name, arg]);
     }
 
-    // Send this condition to sentry because it should never happen.
-    // Check any change in the lib or in the fullnode that could cause an impact here.
-    const errMsg = 'Error while getting the argument names of public_methods on blueprint';
-    dispatch(onExceptionCaptured(new Error(errMsg), false));
-
```

Comment:
This can happen if the dApp sends a request for an invalid blueprint id

And we shouldn't handle this in the component

Reply:
And it was yielding one exception per argument, which is bad

Reply:
I just talked to @alexruzenhack and he reminded me that this is also checking if the method exists in the blueprint, not only if the blueprint exists in the fullnode, so we need to also check for that in the lib PR